python/ray/data/read_api.py

```diff
@@ -2364,6 +2592,14 @@ def from_spark(
         parallelism: The amount of parallelism to use for the dataset. If
             not provided, the parallelism is equal to the number of partitions of
             the original Spark DataFrame.
+        concurrency: The maximum number of Ray tasks to run concurrently. Set this
+            to control number of tasks to run concurrently. This doesn't change the
+            total number of tasks run or the total number of output blocks. By default,
+            concurrency is dynamically decided based on available resource.
+        override_num_blocks: Override the number of output blocks of read tasks. By
+            default, the number of output blocks is dynamically decided based on
+            input data size and available resource. You should not need to manually
+            set this value in most cases.
```

Comment:
these are not used for `from_spark` (same for a few other "from external DB" APIs)

Reply:
Updated.